inc/cookie/namespace.php

```diff
@@ -18,7 +20,7 @@ function is_enabled() : bool {
  */
 function on_login_form() {
 	?>
-	<p><a href="<?php echo esc_url( get_authorize_url() ); ?>">Log In with Delegated Auth</a></p>
+	<p><a href="<?php echo esc_url( get_authorize_url() ); ?>"><?php echo esc_html( HM_DELEGATED_AUTH_LOGIN_TEXT ); ?></a></p>
```

Comment:
```php
// wp-config.php
define( 'HM_DELEGATED_AUTH_LOGIN_TEXT', false );
```

I'd probably do a stronger sanity check to ensure the defined value is valid, maybe `defined() && is_string()`, and fall back to the default if nothing will display.

Reply:
Good call. I've updated that now!